modules/ROOT/pages/deploying-to-cloudhub.adoc

```diff
@@ -129,33 +129,35 @@ image::applicationfile.png[ApplicationFile]
 
 === Runtime Version
 
-Using the drop-down menu, select the *Runtime version* to which you to deploy:
+Using the drop-down menu, select the *Runtime version* to which to deploy:
 
 image::runtimeversion2.png[RuntimeVersion2]
 
 [IMPORTANT]
 ====
-* Ensure that the runtime version is the same Mule version used to develop your application. For example, if you use the value 3.5.0 and your application uses the new HTTP connector introduced in Mule 3.6 and newer, your application won't deploy and the log will contain errors.
-* To see what runtime version you set when you deployed your application, click the *Applications* tab and click your application. In the expansion view on the right, click *Manage Application*. Click *Settings* to see the deployment values.
+* Ensure that the runtime version is the same Mule version used to develop your application. For example, if you use the value 3.5.0 and your application uses the new HTTP connector introduced in Mule 3.6 and later, your application won't deploy and the log will contain errors.
+* To see what runtime version you set when you deployed your application:
+. Click the *Applications* tab and click your application.
+. In the expansion view on the right, click *Manage Application*.
+. Click *Settings* to see the deployment values.
 ====
 
-
-Once deployed and running, if any security patches, OS updates, or critical bug fixes are released for the selected runtime version, then you will be prompted about this change. You will be able to control exactly when each update is applied. If you take no action, updates will be applied automatically for you after 30 days to ensure your applications run with the latest security patches.
+Once the application is deployed and running, if any security patches, OS updates, or critical bug fixes are released for the selected runtime version, then you are prompted about this change. If you take no action, updates are applied automatically after 30 days to ensure your applications run with the latest security patches.
 
 image::deploying-to-cloudhub-update.png[]
 
 
 === Worker Sizing
 
-After deploying your application, you can allocate the amount and size of the worker associated with your application. On each application, workers are responsible for executing your application logic.
+After deploying your application, you can allocate the amount and size of the worker associated with your application. On each application, workers are responsible for executing the application logic.
```

Comment:
I think "number of workers and size of each worker" instead of "amount" will work better here? ("amount of the worker" isn't what they meant, I think"

Reply:
Yes, I agree. I was going to change it and then second guessed myself.